gtk-3.0/gtk-widgets.css

```diff
@@ -2880,6 +2880,31 @@ button.destructive-action:active:hover,
     border-color: shade (@error_color, 0.85);
 }
 
+button.destructive-action:focus,
+.titlebar button.destructive-action:focus,
+.destructive-action.button:focus,
+.titlebar .destructive-action.button:focus {
+    background-image:
+        linear-gradient(
+            to bottom,
+            shade (
+                @error_color,
+                1.4
+            ),
+            shade (
+                @error_color,
+                1.1
+            )
+        );
+    box-shadow:
+        inset 0 1px 0 0 alpha (@bg_highlight_color, 0.3),
```

Comment:
Since your insets are over a background color that you control, it doesn't make sense to use @bg_highlight_color here :p only on the outsets